lib/community_api_client.js

```diff
@@ -0,0 +1,19 @@
+const axios = require("axios");
+
+exports.getServiceUserDetailsByCRN = async function (crn, authToken) {
+  try {
+    const response = await axios.get(
+      `http://localhost:8096/secure/offenders/crn/${crn}/all`,
```

Comment:
What do you think about adding a section in the readme about booting these services up on these ports? (Just in case we'll have to boot it up in a hurry)

Reply:
Sounds good - I put it in the commit message, but our README could definitely do with some love.